xline/src/server/watch_server.rs

```diff
@@ -48,25 +94,30 @@ struct WatchHandle {
     watcher_map: HashMap<WatchId, Watcher>,
     /// Next available `WatchId`
     next_id: WatchId,
+    /// Stop tx
+    stop_tx: mpsc::Sender<()>,
```

Comment:
If the messages transferred in the `stop` channel are meaningless, why don't we use a rendezvous channel here? Like `std::sync::mpsc`, `flume` supports 'rendezvous' channels, a bounded queue with a maximum capacity of zero will block senders until a receiver is available to take the message.

Reply:
done